utils/python/CIME/case_run.py

```diff
@@ -393,17 +209,21 @@ def case_run(case):
     lid = time.strftime("%y%m%d-%H%M%S")
     os.environ["LID"] = lid
 
-    save_timing_setup_acme(case, lid)
+    save_prerun_provenance(case)
 
     for _ in range(data_assimilation_cycles):
```

Comment:
Good point - I don't think that we've considered interaction between the data assimilation cycle and timing code.